### PR TITLE
Added the Readme details for the repo

### PR DESCRIPTION
**Pull request by isamozdran**
Added general information on the project build; 
Listed the required tools;
Please, review for anything that has to added (like badges, awards, or any requirements for contributing)

### CODE REVIEW

**Review by AlexNDRmac — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,13 +1,161 @@
...
+Choose the type of download for your document:
+* `PdfOriginal` - download a document in a state it's been when uploaded to SignNow, before any changes
+* `PdfCollapsed` - download a document in PDF file format
+* `ZipCollapsed` - download a document in ZIP archive
+* `PdfWithHistory` - download a document with its history, a full log of changes on a separate page.
+
+```csharp
+/// using `documentId` from the Upload document step
+
+var downloadPdf = documentService.DownloadDocumentAsync(documentId, downloadType.PdfCollapsed).Result;
```
Comment (IvanBil) [inline README.md:121]: need details how to actually save the file.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,161 @@
...
+
+var pdfFilePath = "./file-path/document-example.pdf";
+var pdfFileName = "document-example.pdf";
+
+/// using token from the Authorization step
+var documentService = new DocumentService(token);
+
+using (var fileStream = File.OpenRead(pdfFilePath))
+{
+    var uploadResponse = documentService.UploadDocumentAsync(fileStream, pdfFileName, default).Result;
```
Comment (IvanBil) [inline README.md]: why "default" in the last param?

**Review by IvanBil — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AlexNDRmac) [reply README.md:121]: added

File: README.md (hunk 2 above)
Comment (AlexNDRmac) [reply README.md]: explicit interface usage. Removed optional param to simplify example

**Review by IvanBil — APPROVED** [review]

**Pull request merged**